docs/qt-removal/ARCHITECTURE_DIAGRAM.md

```
@@ -0,0 +1,380 @@
+# OpenMS Architecture & Qt Dependency Map
+
+## Current Architecture (Before)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)